### DISCUSSION TITLE
Which of the issue trackers for the Pipeline should we use?

### DISCUSSION THREAD

**Discussion by GeorgeKerscher** [daisy/pipeline#676] [Q&A] [answered]
I used the DAISY Pipeline App to convert a dtbook file to HTML, and all headings were mapped to H1. There were h2 and h3 in the dtbook file, but the resulting HTML file did not preserve the structure.

I view this as a bug, and I found three DAISY Pipeline issue trackers. One is the DAISY Pipeline issue tracker, the other is for Pipeline 2, and the other is for the DAISY Pipeline App UI. This seems confusing, but I want to submit the bug described above.

**Comment by bertfrees** [comment, marked as answer]
Hi George. The recommended place to report bugs is https://github.com/daisy/pipeline/issues. Anything can be reported there. It can be related to the desktop app or to Pipeline 1.
